raster-foundry/raster-foundry
# Add layer view navbar component

## Overview

This PR adds a component for the navigation bar when in the new project layer view.

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Demo

<img width="1519" alt="screen shot 2019-02-04 at 6 15 40 pm" src="https://user-images.githubusercontent.com/16109558/52243941-5b6d8900-28a9-11e9-8b67-13b3f8d0dc97.png">

### Notes

For project layer and its related view (such as color corrections, scene browse), we show detailed navigations. For project analyses and its related pages, we just show the navigation for getting to the project itself. We will come back to project analyses navigations in a future PR.


## Testing Instructions

 * Go through all project layer pages according to the routes definition and check if the navigations make sense

Closes #4536
